www/class/centreon-clapi/centreon.Config.Poller.class.php

```diff
@@ -425,12 +429,12 @@ public function pollerGenerate($variables, $login, $password)
      * Move configuration files to servers
      * @param unknown_type $variables
      */
-    public function cfgMove($variables)
+    public function cfgMove($variables = null)
     {
-        global $pearDB, $pearDBO;
+        global $pearDB, $pearDBO, $dependencyInjector;
```

Comment:
Don't use the DI like that. Inject the necessary class instance directly into the method.

Reply:
I found a better way than we discuss yesterday. I haven't noticed we called the DI into the constructor of the class so I've directly use as $this->dependencyInjector